uwp/Pdf-Viewer/Concepts-and-Features/Working-with-free-text-callout-annotation.md

```diff
@@ -0,0 +1,212 @@
+
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+pdfViewer.FreeTextCalloutAnnotationSettings.Opacity = 0.5f;
+
+{% endhighlight %}
+
+### Setting the default minimum width and minimum size
+
+By using the [`MinWidth`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.PdfViewerFreeTextCalloutAnnotationSettings.html#Syncfusion_Windows_PdfViewer_PdfViewerFreeTextCalloutAnnotationSettings_MinWidth) and [`MinHeight`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.PdfViewerFreeTextCalloutAnnotationSettings.html#Syncfusion_Windows_PdfViewer_PdfViewerFreeTextCalloutAnnotationSettings_MinHeight) properties, you can set the minimum width and height of the free text callout annotation for resizing.
+
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+pdfViewer.FreeTextCalloutAnnotationSettings.MinWidth= 100;
+pdfViewer.FreeTextCalloutAnnotationSettings.MinHeight= 100;
+
+{% endhighlight %}
+
+### Enable a free text input box with automatic resizing
+
+By using the [`AutoSizeFreeTextInputBox`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.PdfViewerFreeTextCalloutAnnotationSettings.html#Syncfusion_Windows_PdfViewer_PdfViewerFreeTextCalloutAnnotationSettings_AutoSizeFreeTextInputBox) property, you can enable or disable the automatic resizing of the free text input box.
+
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+pdfViewer.FreeTextCalloutAnnotationSettings.AutoSizeFreeTextInputBox= true;
+
+{% endhighlight %}
+
+### Setting the default callout line segment count
+
+You can set the default callout line segment count of the free text callout annotations by using the [`CalloutLineSegmentCount`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.PdfViewerFreeTextCalloutAnnotationSettings.html#Syncfusion_Windows_PdfViewer_PdfViewerFreeTextCalloutAnnotationSettings_CalloutLineSegmentCount) property. By default, the free text callout annotation is added with two line segments.
+
+{% highlight c# %}
+
+SfPdfViewerControl pdfViewer = new SfPdfViewerControl();
+pdfViewer.FreeTextCalloutAnnotationSettings.CalloutLineSegmentCount = CalloutLineSegmentCount.One;
+
+{% endhighlight %}
+
+### Changing the properties of a selected free text callout
+
+You can change the properties of a selected free text callout annotation or remove it by right clicking it and choosing the desired property from the displayed options.
+ 
+ ![customtoolbarimage](images/image7.png)
+
+## Detecting the movement of free text callout annotations
+
+The movement made in free text callout annotations can be detected using the `FreeTextCalloutAnnotationMoved` event. The property values before and after the change can be obtained using the [`FreeTextCalloutAnnotationMovedEventArgs`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.FreeTextCalloutAnnotationMovedEventArgs.html) parameter of the event’s handler.
```

Comment:
The movement made in a free text callout annotations can be detected using the `FreeTextCalloutAnnotationMoved` event. The property values before and after the change can be obtained using the [`FreeTextCalloutAnnotationMovedEventArgs`](https://help.syncfusion.com/cr/uwp/Syncfusion.Windows.PdfViewer.FreeTextCalloutAnnotationMovedEventArgs.html) parameter of the event’s handler.